lib/Sema/TypeCheckStmt.cpp

```diff
@@ -2056,7 +2054,6 @@ TypeCheckFunctionBodyRequest::evaluate(Evaluator &evaluator,
   // Class constructor checking.
   if (auto *ctor = dyn_cast<ConstructorDecl>(AFD)) {
     if (auto classDecl = ctor->getDeclContext()->getSelfClassDecl()) {
-      checkActorConstructor(classDecl, ctor);
```

Comment:
RE: my comment earlier about `TypeChecker::checkDistributedActor`

I think this might have been the only reference to `checkActorConstructor`, so it should be safe to delete that and possibly the function it calls?

Reply:
Yup, removed now!